pkg/crypto/internal/groth16/bls12381/proof/proof.go

```diff
@@ -0,0 +1,55 @@
+package proof
```

Comment:
I think it's better no to separate `proof` and `verificationKey`(wrong naming also), but join them in one package `internal/groth16/bls12-381/` for example, then use renaming to not collide with `kilic/bls12-381`.